homeassistant/components/awair/strings.json

```
@@ -1,29 +1,49 @@
 {
   "config": {
     "step": {
-      "user": {
-        "description": "You must register for an Awair developer access token at: https://developer.getawair.com/onboard/login",
+      "cloud": {
+        "description": "You must register for an Awair developer access token at: {url}",
         "data": {
           "access_token": "[%key:common::config_flow::data::access_token%]",
           "email": "[%key:common::config_flow::data::email%]"
         }
       },
+      "local": {
+        "data": {
+          "host": "Hostname or IP address of the device"
```

From what I recall, Awair mentions only IP addresses in their documentation. We should do the same:
```suggestion
          "host": "[%key:common::config_flow::data::ip%]"
```